### PR TITLE
Added Watari the Ninja Boy

### PR DESCRIPTION
**Pull request by Eugene208029488**
The link looks good.  Please proceed to merge.  @casey-collab

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you review this PR please?

File: english/watari/watari.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The story [Watari] (https://www.youtube.com/watch?v=AiTTQ5EW0Ik) the ninja boy.
```
Comment (casey-collab) [inline english/watari/watari.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Eugene208029488** [comment]
The link looks good.  Pls proceed to merge the code.  Thanks

**Pull request closed without merging**